tests/integration/models/os.spec.coffee

```diff
@@ -235,6 +241,15 @@ describe 'OS model', ->
 				promise = balena.models.os.getSupportedVersions('foo-bar-baz')
 				m.chai.expect(promise).to.be.rejectedWith('No such device type')
 
+		describe 'given a user that supports ESR', ->
+			# TODO: This should be production plan user
```

Comment:
@thgreasi there is also a paid user envvar, but for ESR we only need production or enterprise plan users. shall we create another instance of a paid user, or shall we make sure that the paid user passed is actually on a production/enterprise plan?

Reply:
@sradevski The paid user that we currently have is for a $0 test plan. Having a test user on a production/enterprice plan probably needs some discussion about the analytics & billing implications.

We could definitely have such a user on staging and run the respective tests when the appropriate env vars are available.

Reply:
@thgreasi if we can enable ESR for the test plan, then that would work as well. Is that possible?

Reply:
I haven't seen the API code for the ESR feature, but that should work as well.